product_ux/i18n/es.po

```diff
@@ -0,0 +1,40 @@
+# Translation of Odoo Server.
+# This file contains the translation of the following modules:
+# * product_usability
+# 
+# Translators:
+# Juan José Scarafía <scarafia.juanjose@gmail.com>, 2018
+# Javier Diez <jd@adhoc.com.ar>, 2018
+msgid ""
+msgstr ""
+"Project-Id-Version: Odoo Server 9.0c\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2018-03-19 13:42+0000\n"
+"PO-Revision-Date: 2018-03-19 13:42+0000\n"
+"Last-Translator: Javier Diez <jd@adhoc.com.ar>, 2018\n"
+"Language-Team: Spanish (https://www.transifex.com/adhoc/teams/46451/es/)\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: \n"
+"Language: es\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+
+#. module: product_usability
```

Comment:
@jjscarafia  we were talking about what to do with translation when we migrated. We do not have a steps to follow. Will be good to discuss in the next meeting